### DISCUSSION TITLE
Welcome to qgis-gnavs-plugin Discussions!

### DISCUSSION THREAD

**Discussion by b-lack** [b-lack/search-find-qgis-plugin#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wein-franke** [comment]
Just an idea...

I am using QGIS on a tablet that is part of a guidance system for agricultural machines, which is baesd on Windows. I own an additional GNSS Receiver (based on UBlox ZED F9P) that I can connect over bluetooth. To find points without the need to detach the tablet from the guidance system I am thinking about a voice-guidance. Maybe the frequency of a tone shows the distance and the direction is told in words? Or take the frequency of the repetition of the tone as a measure for accurancy and the freqency of the tone itself to find the direction (like the game "hot and cold" that children play.)
Would that be possible without much efforts?

Nur mal so als Idee...

Ich verwende QGIS auf einem Windows-Tablet, das vorrangig als Rechner für ein Lenksystem für Landmaschinen dient. Zusätzlich zum GNSS Receiver des Lenksystems kann ich das Tablet auch mit einem externen Receiver auf einem Besenstiel über Bluetooth verbinden.
Um einen einzelnen Markstein zu suchen, ohne das Tablet vom Lenksystem abbauen zu müssen, denke ich gerade darüber nach, ob man Entfernung und Richtung zum Ziel nicht auch als Tonsignal ausgeben könnte? Beispielsweise die Tonhöhe oder die Wiederholrate eines Piepstones als Maß für die Entfernung und eine Ansage für die Richtung? Oder eben eine veränderliche Tonhöhe wenn man in die richtige oder falsche Richtung geht, wie beim Kinderspiel, bei dem jemand etwas suchen muss und von den anderen Kindern den Hinweis kalt oder warm bekommt....

Oder wäre eine automatische Ansage analog zur Vermessung mit dem Theotoliten möglich, die ausgehend vom Startpunkt der Messung, den verbliebenen Abstand mit "x cm näher/weiter, y cm rechts/links" ansagt?

Wäre so etwas ohne großen Aufwand machbar?

**Reply by b-lack** [reply]
Für den speziellen Anwendungsfall sehe ich den Nutzen. Der ist aber sehr speziell und macht meiner Meinung nach nur Sinn, wenn man QGIS nicht während der Navigation offen hat oder nicht aufs Display schauen möchte, wenn der Lautsprecher aktiv und laut genug ist diesen zu verstehen.

Da keine Kompass-Funktion eingebaut ist, wäre auch die Funktion "nach links/rechts" so gar nicht umsetzbar.

Die Idee finde ich super. Ob der Nutzen den Aufwand rechtfertigt ist zu diskutieren.

**Reply by wein-franke** [reply]
Anstelle der Lautsprecher hätte ich zwar an Kopfhörer/Earbuds gedacht. Aber der Aufwand ist für die nur gelegentliche Nutzung sicherlich enorm. Das sehe ich ein.
